backend/plugin/db/mysql/sync.go

```diff
@@ -266,6 +266,11 @@ func (driver *Driver) SyncDBSchema(ctx context.Context) (*storepb.DatabaseSchema
 			} else {
 				column.DefaultValue = &storepb.ColumnMetadata_Default{Default: &wrapperspb.StringValue{Value: defaultStr.String}}
 			}
+		} else {
+			// TODO(zp): refactor column default value.
+			if strings.Contains(extra, "auto_increment") {
```

Comment:
Consider case sensitivity?

Reply:
I found that `auto_increment` is always lowercase. https://dev.mysql.com/doc/refman/8.0/en/information-schema-columns-table.html. But I will update it.

Reply:
Yup, just in case if not.